katz/src/main/kotlin/katz/Id.kt

```diff
@@ -0,0 +1,31 @@
+package katz
+
+class Id<out A>(val value: A): HK<Id.F, A> {
+
+    class F
```

Comment:
if compiler does not complain, should be an object, the worst case it's only one instance of F shared across all instances of the type.